q2_quality_filter/_examples.py

```diff
@@ -0,0 +1,27 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+m_p_base = 'https://data.qiime2.org/usage-examples/moving-pictures/'
+demuxed_seqs_url = m_p_base + 'demux.qza'
```

Comment:
```suggestion
demuxed_seqs_url = 'https://data.qiime2.org/usage-examples/moving-pictures/demux.qza'
```
I don't think this abstraction is worth it, and we've been just using the entire URL in our other examples, so may as well keep things consistent 🤷‍♀️